### PR TITLE
New private Google group for UX WG leads

### PR DESCRIPTION
**Pull request by aliok**
# Changes

- I want to create a new mail alias for the UX WG leads. They're gonna sign up to some systems (Figma, etc) and they want to share their username and password for consolidation.
- So, obviously, this group needs to be private.

### CODE REVIEW

File: groups/wg-ux/groups.yaml
Comment (aliok) [inline groups/wg-ux/groups.yaml:1]: I couldn't find if there's a setting for "private". But I want this group to be private. See PR description above.

File: groups/wg-ux/groups.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    description: |-
+      UX WG email
+    settings:
+      WhoCanPostMessage: "ANYONE_CAN_POST"
+      ReconcileMembers: "true"
+    owners:
+      - zainabhusain@ocadu.ca
+      - mariana.mejia@ocadu.ca
+      - leoli@redhat.com
+      - cmurray@redhat.com
```
Comment (aliok) [inline groups/wg-ux/groups.yaml:14]: /hold

@mmejia02 @zainabhusain227 @Leo6Leo @Cali0707 

Are these email addresses good?

Comment (Cali0707) [reply groups/wg-ux/groups.yaml:14]: These seem to match the emails used on the calendar invite for the meetings so I think it's correct

**Review by upodroid — APPROVED** [review]
This configuration creates a private group and membership changes need to be done via  Git.

**Pull request merged**